docs.bak/content/en/models/telegraph-chat.md

```diff
@@ -59,7 +59,17 @@ Retrieves the chat member count from telegram
 ```php
 /** @var \DefStudio\Telegraph\Models\TelegraphChat $telegraphChat */
 
-$telegraphChat->info(); /* 42 */
+$telegraphChat->memberCount(); /* 42 */
+```
+
+### `memberInfo()`
+
+Retrieves the chat member count from telegram
```

Comment:
@MarioGattolla could you fix this docs description? I think it doesn't returns just the member count